packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -119,6 +119,8 @@ The `Accept-Language` HTTP header advertises which languages the client is able
 
 ## Device Fingerprint
 
+<ApiLifecycle access="deprecated" />
```

Comment:
Instead of marking this as deprecated, if device fingerprint is not used in sign-on policy evaluation we should delete the highlighted sentence in the following section.

![Screen Shot 2021-01-20 at 11 19 23 AM](https://user-images.githubusercontent.com/6888475/105224078-90645c00-5b11-11eb-8fe8-bcf9195c3b9d.png)

My understanding is that we still use X-Device-Fingerprint for trusted apps but the usage by Okta is narrowed down to just sending new device email notifications.